### DISCUSSION TITLE
[Suggestion] Fuel/System Balance

### DISCUSSION THREAD

**Discussion by SabreMania** [tsunamayo/Starship-EVO#4163] [Suggestions]
**The Long Part**

Greetings! With the most recent addition of Fuel to the game, a lot of things changed. We finally have an extra piece of 'hard' data to work with on the long path to game balance. But this one is unique, in the sense that it is not something we never run out of like energy or shields in previous versions of the game, in fact it comes and places a hard limit on how often each of those can happen, which brings in all sorts of interesting implications to a vessel's design that we didn't have to work with before. It's not surprising that we've reached this point, but it's very exciting!

Now, I personally haven't touched the numbers yet or done any tests, but this isn't aimed at them. I'm not here to say 'Buff/Nerf fuel consumption rate for the reactors and the thrusters' or something like that. Rather, I'd like to elaborate on a very simple concept concerning something that hasn't been brought to light in a little while.

A few months ago, the concept of systems destroying adjacent systems was introduced. Simultaneously, if I recall, you (Tsuna) mentionned that it was your desire that people keep their systems under one single stretch as much as possible. This of course is problematic since you gain nothing from doing that, and on top of it there's the risk associated with packing your systems up where you simply lose a huge chunk of it from a single lucky shot. All in all, ship design, so far, had been focused around separating your systems as much as possible and keeping them small, but numerous.

**Too Long, didn't read. Get to the point.**

Alright, so essentially, here's what I suggest to remedy to this problem, now that Fuel has been added and that every system requires it to function.

Making it so that the more a system is stretched, from a single block, the more fuel-efficient it is, or the less fuel per unit of power it consumes. 

The Pros:

> Ship designers will now have a real reason to stretch a single system as much as possible, rather than place individual blocks separated by, well, one block

> It adds depth to the building gameplay, as you must now carefully weigh how much you want to stretch your system before you place down another and stretch that one, in order to compartimentalize your ship and avoid catastrophic damage.

The Cons:

> It partially eats away at creativity, since there is now a better way to build one way over another (However, being a game rather than a ship designing program, this is not so undesirable)

> Small ships, such as fighters and frigates, may suffer from this, as you can only stretch a block so much before you run out of space, which is particularly true when your ship is only a dozen or so meters long and that most of it's width is just wingspace. 

This is just my humble little suggestion, and actual game balance may not be around the corner, i'm not in your head after all, but if it can help you figure something out, it may as well be where you can see it, rather than being lost under millions of messages on a forum you don't go on.

**Comment by tsunamayo** [comment]
@SabreMania Hi, indeed this is a segment of the game where what was created no longer apply because of the tech switch.
Tech is now much more optimized, so there is little cost of having several reactors instead of a single one.
But I am open to switch like that further down the road if it is necessary, but I will only do it if it is something that feels necessary after some playtest. 
Cheers
